content/docs/command-reference/remote/modify.md

```diff
@@ -250,27 +250,71 @@ For more information on configuring Azure Storage connection strings, visit
 
 Please check out
 [Setup a Google Drive DVC Remote](/doc/user-guide/setup-google-drive-remote) for
-a full guide on configuring Google Drives for use as DVC remote storage,
-including obtaining the necessary credentials, and how to form `gdrive://` URLs.
+a full guide on using Google Drive as DVC remote storage, including how to form
+`gdrive://` URLs, and high-performance/enterprise setups.
 
 - `url` - remote location URL.
```

Comment:
put link to how URL is formed right here

Reply:
Hmmm, GDrive URL formation is mentioned and linked in in the paragraph above, seems overkill to repeat it here?

Reply:
that's the whole point ) no one reads long descriptions (unless you really need to clarify some behavior) :) It's better to remove it from that paragraph (at least mention that the link explain URLs as well) ... or repeat it. The way I would read is a I would start with the list of params and it's really better to be self-descriptive more or less.

Reply:
OK, refactored with shorter p about full guide, and additional links to guide sections in params.